SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -598,6 +599,20 @@ int64 USpatialActorChannel::ReplicateActor()
 
 	ReplicationBytesWritten = 0;
 
+	if (!bCreatingNewEntity && NeedOwnerInterestUpdate())
+	{
+		AActor* HierarchyRoot = SpatialGDK::GetHierarchyRoot(Actor);
+		Worker_EntityId OwnerId = NetDriver->PackageMap->GetEntityIdFromObject(HierarchyRoot);
+
+		if (HierarchyRoot == nullptr || OwnerId != SpatialConstants::INVALID_ENTITY_ID)
```

Comment:
What can happen between here and calling SetNeedOwnerInterestUpdate to change the owner ID?

Reply:
I setup it this way to avoid sending an update if the owner is still not ready.

Reply:
What I mean is, if we read the owner ID here, why do we need the bOwnerReady flag?
The flag seems to just be a checking the same value.